eclair-core/src/main/resources/reference.conf

```
@@ -90,6 +90,7 @@ eclair {
     // maximum local vs remote feerate mismatch; 1.0 means 100%
     // actual check is abs((local feerate - remote fee rate) / (local fee rate + remote fee rate)/2) > fee rate mismatch
     max-feerate-mismatch = 1.56 // will allow remote fee rates up to 8x bigger or smaller than our local fee rate
+    close-on-offline-feerate-mismatch = true // if false eclair will not close offline channel on feerate mismatch
```

```suggestion
    close-on-offline-feerate-mismatch = true // if false eclair will not close offline channels on feerate mismatch
```

I don't think this comment adds much information. How about the good old "do not change this unless you know what you are doing" instead?

---

I tried my best :) but in fact this config key should be self explanatory, will add.